services/models/health.go

```diff
@@ -4,12 +4,15 @@ package models
 type Health struct {
 	Envoy              EnvoyHealth        `json:"envoy"`
 	DeploymentStatuses []DeploymentStatus `json:"deploymentStatuses"`
+	Requests           RequestHealth      `json:"requests"`
+	DeploymentsFetched bool               `json:"-"`
```

Comment:
This 'json:"-"' means that this field is not serialized in the json ?
If so, is this field auxiliar ?

Reply:
yes it's the golang way to ignore it : https://golang.org/pkg/encoding/json/#Marshal
What do you mean by auxiliar?